ntcore/src/generate/java/EntryImpl.java.jinja

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.networktables;
+
+/** NetworkTables {{ TypeName }} implementation. */
+@SuppressWarnings("PMD.ArrayIsStoredDirectly")
```

Comment:
Why is this suppression needed? And can't it be scoped smaller?

Reply:
It’s because in the array case (eg double[]), the defaultValue passed to the constructor is simply stored in the member variable instead of being deep-copied.